api/docker-compose.yml

```diff
@@ -7,3 +7,4 @@ elasticsearch:
     - "http.host=0.0.0.0"
     - "transport.host=0.0.0.0"
     - "cluster.name=wellcome"
+    - "logger.level=DEBUG"
```

Comment:
I think so. Trying to debug what's going on when a test fails is pretty much useless without it.